src/eins-timer.c

```diff
@@ -0,0 +1,210 @@
+/* Copyright 2021 Endless OS Foundation LLC. */
```

Comment:
I believe this code comes in part from eos-payg, so its copyright date should be earlier.

Reply:
I prefer the naming `eins-boottime-source.[ch]`.

Reply:
ok